
2022-10-25 19:07:48.151 - [Debug] [Connection] Attacker connected: 54.38.97.156 | Client Identification: SSH-2.0-Go
2022-10-25 19:07:48.154 - [Debug] [Auth] Attacker 78.189.25.224 trying to authenticate with 'password'
2022-10-25 19:07:48.155 - [Debug] [Auto Access] Attacker: 78.189.25.224, Threshold: 1, Attempts: 1
2022-10-25 19:07:48.155 - [Debug] [Auto Access] Compromising the honeypot
2022-10-25 19:07:48.155 - [Debug] [Auto Access] Adding the following credentials: 'root:1'
2022-10-25 19:07:48.399 - [Debug] [Auto Access] Auto-access is now disabled for the remainder of this MITM server instance
2022-10-25 19:07:48.407 - [Debug] [Auth] Valid credentials - Password Authentication
2022-10-25 19:07:48.407 - [Debug] [LXC] Attempting to connect to the honeypot: 10.0.3.12
2022-10-25 19:07:48.542 - [Debug] [Auth] Attacker 223.171.91.191 trying to authenticate with 'publickey'
2022-10-25 19:07:48.548 - [Debug] [Auth] Attacker authentication error: Publickey authentication failed
2022-10-25 19:07:48.550 - [Debug] [Auth] Attacker: 223.171.91.191 has so far made 1 attempts. Remaining: 5 attempts
2022-10-25 19:07:48.637 - [Debug] [LXC-Auth] Attacker authenticated and is inside container
2022-10-25 19:07:48.909 - [Debug] [Auth] Attacker 54.38.97.156 trying to authenticate with 'none'
2022-10-25 19:07:48.909 - [Debug] [Auth] Attacker authentication error: No authentication method provided
{"level":"error","message":"Forever detected script was killed by signal: SIGKILL"}
(node:735566) Warning: Accessing non-existent property 'padLevels' of module exports inside circular dependency
(Use `node --trace-warnings ...` to show where the warning was created)
Started with the following options:
{
  debug: true,
  mitmIp: '10.0.3.1',
  autoAccess: true,
  autoAccessCache: 5000,
  maxAttemptsPerConnection: 6,
  sshServerIdentifier: 'SSH-2.0-OpenSSH_6.6.1p1 Ubuntu-2ubuntu2',
  containerMountPathPrefix: '/var/lib/lxc',
  containerMountPathSuffix: 'rootfs/',
  loggingAttackerStreams: '/home/student/MITM/logs/session_streams',
  loggingAuthenticationAttempts: '/home/student/MITM/logs/authentication_attempts',
  loggingLogins: '/home/student/MITM/logs/logins',
  loggingKeystrokes: '/home/student/MITM/logs/keystrokes',
  containerName: 'DATABASE_2',
  containerIp: '10.0.3.212',
  mitmPort: 1001,
  autoAccessFixed: 1
}
2022-10-25 19:09:01.589 - [Info] MITM Version: 2
2022-10-25 19:09:01.591 - [Info] Auto Access Enabled: true
2022-10-25 19:09:01.591 - [Debug] [Init] Auto Access Theshold Achieved: false
2022-10-25 19:09:01.599 - [Info] SSH man-in-the-middle server for 10.0.3.212 listening on 10.0.3.1:1001
2022-10-25 19:09:01.962 - [Debug] [Connection] Attacker connected: 85.53.55.133 | Client Identification: SSH-2.0-Go
2022-10-25 19:09:02.050 - [Debug] [Connection] Attacker connected: 31.165.5.155 | Client Identification: SSH-2.0-Go
2022-10-25 19:09:02.165 - [Debug] [Connection] Attacker connected: 20.127.87.36 | Client Identification: SSH-2.0-Go
2022-10-25 19:09:02.246 - [Debug] [Auth] Attacker 20.127.87.36 trying to authenticate with 'none'
2022-10-25 19:09:02.247 - [Debug] [Auth] Attacker authentication error: No authentication method provided
2022-10-25 19:09:02.327 - [Debug] [Auth] Attacker 20.127.87.36 trying to authenticate with 'publickey'
2022-10-25 19:09:02.329 - [Debug] [Auth] Attacker authentication error: Publickey authentication failed
2022-10-25 19:09:02.330 - [Debug] [Auth] Attacker: 20.127.87.36 has so far made 1 attempts. Remaining: 5 attempts
2022-10-25 19:09:02.358 - [Debug] [Connection] Attacker closed the connection
2022-10-25 19:09:02.390 - [Debug] [Connection] Attacker connected: 20.127.87.36 | Client Identification: SSH-2.0-Go
